src/renderer/components/message/MessageList.tsx

```diff
@@ -236,20 +236,20 @@ export default function MessageList({ accountId, chat, refComposer }: Props) {
         log.debug('onScroll: Lets try loading messages from both ends')
         setTimeout(() => fetchMoreTop(), 0)
         setTimeout(() => fetchMoreBottom(), 0)
-        Event?.preventDefault()
-        Event?.stopPropagation()
+        evt?.preventDefault()
+        evt?.stopPropagation()
         return false
       } else if (distanceToTop < 200) {
         log.debug('onScroll: Scrolled to top, fetching more messages!')
         setTimeout(() => fetchMoreTop(), 0)
-        Event?.preventDefault()
-        Event?.stopPropagation()
+        evt?.preventDefault()
```

Comment:
on other places we use only `ev` as abbreviation.

Reply:
okay so I will change it to `ev` here as well.